packages/drivers/odsp-driver-definitions/src/resolvedUrl.ts

```diff
@@ -95,6 +95,11 @@ export interface ShareLinkInfoType {
 	 * permission then this link can be redeemed for the permissions in the same network call.
 	 */
 	sharingLinkToRedeem?: string;
+
+	/**
+	 * If the redeem for the sharing link is non-durable, this flag will be set to true.
```

Comment:
Nit: I think it's probably self-explanatory, but just to be explicit
```suggestion
	 * If the redeem for the sharing link is non-durable, this flag will be set to true.
	 * @defaultValue false
```

Reply:
It would also probably be good to define "non-durable" in the docs (or link to something that does)